src/verifier/SourcifyVerifier.ts

```diff
@@ -26,8 +26,17 @@ export class SourcifyVerifier extends AbstractVerifier implements Verifier {
     await this.checkBoundaries();
 
     const res = await this.sendVerifyRequest(artifact);
-    enforceOrThrow(res.data?.result?.length === 1, `Failed to connect to Sourcify API at url ${SOURCIFY_API_URL}`);
 
+    const verificationStatus = res.data?.result[0]?.status;
+    if (verificationStatus !== 'perfect' || verificationStatus !== 'partial') {
```

Comment:
Won't this always evaluate to `true`?

Reply:
See the edited PR description. This is for the case case when the server returns `200` but `status: null` or something else